doc/connectivity/connectome_extraction.rst

```diff
@@ -138,7 +138,7 @@ And it provides one estimated covariance and inverse-covariance
 
 |
 
-.. currentmodule:: nilearn.group_sparse_covariance
+.. currentmodule:: nilearn.connectome.group_sparse_covariance
```

Comment:
I think that you should be able to have simply nilearn.connectome here, given that GroupSparseCovariance is imported in nilearn.connectome.__init__. Isn't it the case?